features/upgrade/api_auth/upgrade.feature

```diff
@@ -219,11 +220,21 @@ Feature: apiserver and auth related upgrade check
       | o             | jsonpath={.users} |
     And the output should match:
       | system:serviceaccount:test-scc:test-scc |
+    # Below step is to confirm that the pre-upgrade script is executed before post-upgrade check for OCP-29741
+    When I run the :new_project client command with:
+      | project_name | test29741-project |
+    Then the step should succeed
+    And I run the :create_configmap client command with:
+      | name         | test-cm   |
+      | from_literal | key=foo   |
+      | from_literal | value=bar |   
+    Then the step should succeed
```

Comment:
```suggestion
    # This case's post-upgrade script occasionally failed which was hard to debug. It was often found this case's pre-upgrade script was not executed due to whatever reason.
    # Adding a resource at the end of this pre-upgrade and checking it at the beginning of the post-upgrade can help easier debugging.
    When I run the :new_project admin command with:
      | project_name | ocp-29741-pre-upgrade-done |
```

Reply:
In addition, in upgrade, we should use "When I run the :... **admin** command ..." instead of "When I run the :... **client** command ...". Because we hit and fixed failures where the pre-upgrade Runner job for a case used testuser-xxx while the post-upgrade Runner job for the same case used different user testuser-yyy, which is a framework issue.